packages/ui/src/components/tailwind/ConfirmDialog/index.tsx

```diff
@@ -62,15 +64,24 @@ export const ConfirmDialog = ({ title, text, onSubmit, onClose, modalProps }: Co
         PopoverState.hidePopupover()
         onClose?.()
       }}
-      className="w-[50vw] max-w-2xl"
+      className={twMerge(
+        'w-[50vw] max-w-2xl bg-surface-1',
+        isMobile ? 'h-[90dvh] w-[50vw] min-w-[720px] max-w-2xl' : ''
```

Comment:
we can use media queries like `smh` and `lg`

Reply:
done